connectors/src/connectors/notion/temporal/activities.ts

```diff
@@ -1719,10 +1697,17 @@ export async function renderAndUpsertPageFromCache({
   let upsertTs: number | undefined = undefined;
   let skipReason: string | null = null;
 
-  if (renderedPage.length > MAX_DOCUMENT_TXT_LEN) {
+  // compute document length by summing all prefix and content sizes for each section
+  const sectionLength = (section: CoreAPIDataSourceDocumentSection): number =>
```

Comment:
let's move that here: https://github.com/dust-tt/dust/blob/main/types/src/core/data_source.ts#L29